config.md

```diff
@@ -390,20 +405,166 @@ Here is a full example `config.json` for reference.
     "hooks": {
         "prestart": [
             {
-                "path": "/usr/bin/uptime",
+                "path": "/usr/bin/fix-mounts",
                 "args": [
-                    "/usr/bin/uptime"
+                    "fix-mounts",
+                    "arg1",
+                    "arg2"
                 ],
-                "env": []
+                "env": [
+                    "key1=value1"
+                ]
+            },
+            {
+                "path": "/usr/bin/setup-network"
+            }
+        ],
+        "poststart": [
+            {
+                "path": "/usr/bin/notify-start",
+                "timeout": 5
+            }
+        ],
+        "poststop": [
+            {
+                "path": "/usr/sbin/cleanup.sh",
+                "args": [
+                    "cleanup.sh",
+                    "-f"
+                ]
             }
         ]
     },
     "linux": {
+        "devices": [
+            {
+                "path": "/dev/fuse",
+                "type": "c",
+                "major": 10,
+                "minor": 229,
+                "fileMode": 438,
+                "uid": 0,
+                "gid": 0
+            },
+            {
+                "path": "/dev/sda",
+                "type": "b",
+                "major": 8,
+                "minor": 0,
+                "fileMode": 432,
+                "uid": 0,
+                "gid": 0
+            }
+        ],
+        "sysctl": {
+            "net.ipv4.ip_forward": "1",
+            "net.core.somaxconn": "256"
+        },
+        "cgroupsPath": "/myRuntime/myContainer",
         "resources": {
+            "network": {
+                "classID": 1048577,
+                "priorities": [
+                    {
+                        "name": "eth0",
+                        "priority": 500
+                    },
+                    {
+                        "name": "eth1",
+                        "priority": 1000
+                    }
+                ]
+            },
+            "pids": {
+                "limit": 32771
+            },
+            "hugepageLimits": [
+                {
+                    "pageSize": "2MB",
+                    "limit": 9223372036854772000
+                }
+            ],
+            "oomScoreAdj": 100,
+            "memory": {
+                "limit": 536870912,
+                "reservation": 0,
+                "swap": 536870912,
+                "kernel": 0,
+                "kernelTCP": 0,
+                "swappiness": 0
+            },
+            "cpu": {
+                "shares": 0,
+                "quota": 1000000,
+                "period": 500000,
+                "realtimeRuntime": 0,
```

Comment:
This looks like another meaningless value.  I'm not sure how it relates to the other settings, but zero probably means either "never run this" or "there is no limit on this cgroup's continuous CPU access”, neither of which sound like the goal of a user who explicitly specifies this setting.